docker/package/tezos_setup_wizard.py

```diff
@@ -165,15 +165,39 @@ def validate(self, input):
 suppress_warning_text = "TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=YES"
 
 
+def exit_with_confirm(signum, frame):
+    signal.signal(signal.SIGINT, original_sigint)
+
+    try:
+        print()
+        if yes_or_no(
+            "You're about to quit the setup wizard. All the services and processes "
+            "already started will continue to run. Confirm exit? (y/N) ",
+            "no",
+        ):
+            print("Exiting the Tezos Setup Wizard.")
+            sys.exit(1)
+    except KeyboardInterrupt:
+        print()
```

Comment:
I think that we should still have a way to kill everything when exiting via `Ctrl + C`. What do you think about killing all processes here in case one additional signal was sent by the user?